### ISSUE TITLE
Feature Request - Agent create/edit pain points

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7940]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**

Please see my recordings in the Google Drive - https://drive.google.com/drive/folders/1W9sC7COTqRKBxhat3ZqfRmHv_04tQAei?usp=drive_link

**Describe what you're trying to accomplish** 

Save time and make working with agents easier.

**Describe the solution you'd like**

Can we provide a way to remove the big red box of text when creating agents for certain users? Somehow make it so that when I am working on Agents for an entire day I am not spending a quarter of it scrolling past the warnings.

After an agent is created - go straight to the edit page.

I think the reason my name doesn't autofill is because there is a bad duplicate. That seems nuts? Can we ignore bad duplicates and let Arctos pick me when I enter my username?

Would it be possible to return to the same or the next field after TAB to select an agent name?

**Describe alternatives you've considered**

Just keep spending a lot of time scrolling and switching between the keyboard and mouse.

**Additional context**
_Add any other context or screenshots about the feature request here._

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

**Comment by Jegelewicz** [comment]
Here is another pain point. Put names through the Agent prebulkloader and get a bunch of potential matches. Reviewing them is smooshed into this weird block.

![image](https://github.com/user-attachments/assets/cddeb9f4-e955-4cdd-989d-86c79798e294)

**Comment by Jegelewicz** [comment]
@ekrimmel does any of this matter to you?

**Comment by ekrimmel** [comment]
@Jegelewicz, yes! Echoing what Teresa says above, scrolling past the giant red box in order to create a new agent is a pain point for me too. Could probably solve this in a number of ways (make it disappear for some users, open the create agents interface in a new tab vs. a pop-up, etc). Screenshot included for easy visual reference:
<img width="1336" height="971" alt="Image" src="https://github.com/user-attachments/assets/7b5bb2a5-1e6a-408d-ade6-4e1573149602" />
